### PR TITLE
Adding HHT ontology

### PR DESCRIPTION
**Pull request by Brainchain09**

### CODE REVIEW

**Comment by dgarijo** [comment]
@Brainchain09 can you please add yourself as a maintainer in the README file? Or add the github id of the maintainers.
Thanks in advance

File: HHT/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+## What is the HHT Ontology ?
+
+HHT is an ontology to represent historical territories involved in multiple overlaying hierarchies and their evolutions.
+
+Ontology documentation redirection : https://w3id.org/IRIT/MELODI/HHT ->  https://www.irit.fr/recherches/MELODI/ontologies/HHT/
+
+## Maintainer
+William Charles (william.charles@irit.fr)
+Nathalie Aussenac-Gilles (nathalie.aussenac-gilles@irit.fr) ORCID 0000-0003-3653-3223
+Nathalie Hernandez (nathalie.hernandez@irit.fr)
```
Comment (TallTed) [inline HHT/README.md:12]:
```suggestion
## Maintainers
* William Charles (william.charles@irit.fr)
* Nathalie Aussenac-Gilles (nathalie.aussenac-gilles@irit.fr) ORCID 0000-0003-3653-3223
* Nathalie Hernandez (nathalie.hernandez@irit.fr)
```

File: HHT/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Hierarchical Historical Territories HHT
+
+## What is the HHT Ontology ?
+
+HHT is an ontology to represent historical territories involved in multiple overlaying hierarchies and their evolutions.
+
+Ontology documentation redirection : https://w3id.org/IRIT/MELODI/HHT ->  https://www.irit.fr/recherches/MELODI/ontologies/HHT/
```
Comment (TallTed) [inline HHT/README.md:7]:
```suggestion
Ontology documentation redirection: https://w3id.org/IRIT/MELODI/HHT -> https://www.irit.fr/recherches/MELODI/ontologies/HHT/
```

**Comment by Brainchain09** [comment]
> @Brainchain09 can you please add yourself as a maintainer in the README file? Or add the github id of the maintainers. Thanks in advance

I just added my GitHub ID aside my name. Is that what you needed ?

File: HHT/README.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+HHT is an ontology to represent historical territories involved in multiple overlaying hierarchies and their evolutions.
+
+Ontology documentation redirection: https://w3id.org/IRIT/MELODI/HHT -> https://www.irit.fr/recherches/MELODI/ontologies/HHT/
+
+## Maintainers
+* William Charles (william.charles@irit.fr) (GitHub ID : Brainchain09)
+* Nathalie Aussenac-Gilles (nathalie.aussenac-gilles@irit.fr) ORCID 0000-0003-3653-3223
+* Nathalie Hernandez (nathalie.hernandez@irit.fr)
+.
```
Comment (TallTed) [inline HHT/README.md:13]:
```suggestion
```

**Pull request merged**